File: C#/6kyu - A.cs
//Link -> https://www.codewars.com/kata/587731fda577b3d1b0001196

using System;

namespace Kata
{
  public static class Problem
  {
    public static string CamelCase(this string str)  
    {  
        string[] words = (str ?? "").Split();
        string result = string.Empty;

        foreach (string word in words)
        {
            if(word.Length > 1)
            {
               result += Char.ToUpper(word[0]) + word.Substring(1);
            }
            else if(word == "")
            {
               result += "";
            }
            else
            {
               result += Char.ToUpper(word[0]);
            }
        }
            
        return result;
    }
  }
}

File: python/6kyu - A.py
# Link -> https://www.codewars.com/kata/54b42f9314d9229fd6000d9c

def duplicate_encode(word):
    encode = ''
    word = word.lower()

    for i in word:
        x = word.count(i)

        if x > 1:
            encode += ')'
        else:
            encode += '('
    
    return encode
